src/test/test-resize-fs.c

```diff
@@ -0,0 +1,9 @@
+#include "resize-fs.h"
```

Comment:
`SPDX-License-Identifier` line is missing

Reply:
Okay sir.
Please I'd like to also get more comments on how I can make it even better, because if I succeed in getting a merge, it'll be a privilege to have authored a file -albeit a small one- on such a huge project.

Thank you sir.
1. Add SPDX-License Identifier line.

Reply:
the goal is to test the function, so you should start by understand what it does and then write a meaningful test for it

Reply:
> the goal is to test the function, so you should start by understand what it does and then write a meaningful test for it

Exactly sir. The tasks are helping me get more familiar with the code base.


I've added a few more tests sir.

Reply:
The core functionally is still not covered - resizing a filesystem